dummy/src/toga_dummy/widgets/base.py

```diff
@@ -7,9 +7,16 @@ def __init__(self, interface):
         super().__init__()
         self.interface = interface
         self.interface._impl = self
-        self.viewport = None
+        self.container = None
         self.create()
 
+    @property
+    def viewport(self):
+        return self.container
```

Comment:
This has been removed from all the other backends; can it be removed here as well?

Reply:
Yes - that's an oversight on my part.